src/main/webapp/WEB-INF/web.xml

```diff
@@ -1,8 +1,8 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<web-app xmlns="http://java.sun.com/xml/ns/javaee"
+<web-app xmlns="http://xmlns.jcp.org/xml/ns/javaee"
     xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
-    xsi:schemaLocation="http://java.sun.com/xml/ns/javaee http://java.sun.com/xml/ns/javaee/web-app_3_0.xsd"
-    version="3.0">
+    xsi:schemaLocation="http://xmlns.jcp.org/xml/ns/javaee http://xmlns.jcp.org/xml/ns/javaee//web-app_3_1.xsd"
+    version="3.1">
```

Comment:
- [ ] A version change from 3.0 to 3.1 can cause behavioral changes, so you should consider leaving it at 3.0 or writing a migration to 3.1.

Reply:
After considering the incompatibility of the AP server and application with the update, and the support status of the Servlet version in Terasoluna, we decided to maintain the current status.
Therefore, we have reverted the parts of the Servlet schema that have been updated.
https://github.com/terasolunaorg/terasoluna-gfw-web-blank/pull/455/commits/03f53f1bd3d760d2eeceaab9cb1e73e48c3f17d5